src/pages/Title/components/NextSeasons/Stars/Stars.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+//@ts-ignore
+import ReactStars from 'react-rating-stars-component'; // solve the problem with typeScript?
```

Comment:
you should ignore this, this library doesn't support typescript, but we have some solutions from vscode Intelisense: https://prnt.sc/WUeE2hiewg99 that is to add "declare module"

Reply:
![image](https://github.com/GGO-web/Unimay/assets/97807120/96bf750a-af4c-42b2-bc68-52648d8450cd), Is that normal?

Reply:
Yes, because there is no package defined for types for this library
I tried it too